src/algebra/module/equiv.lean

```diff
@@ -357,6 +357,15 @@ protected lemma bijective : function.bijective e := e.to_equiv.bijective
 protected lemma injective : function.injective e := e.to_equiv.injective
 protected lemma surjective : function.surjective e := e.to_equiv.surjective
 
+lemma bijective_to_linear_map : function.bijective e.to_linear_map :=
+  eq.substr (coe_to_linear_map e) e.bijective
+  
+lemma injective_to_linear_map : function.injective e.to_linear_map :=
+  eq.substr (coe_to_linear_map e) e.injective
+
+lemma surjective_to_linear_map : function.surjective e.to_linear_map :=
+  eq.substr (coe_to_linear_map e) e.surjective
```

Comment:
```suggestion
lemma bijective_to_linear_map : function.bijective e.to_linear_map := e.bijective
lemma injective_to_linear_map : function.injective e.to_linear_map := e.injective
lemma surjective_to_linear_map : function.surjective e.to_linear_map := e.surjective
```
simply works due to defeq. So I don't quite see the point of these new lemmas.

Reply:
The reason I introduced those is that I want to be able to use `linear_map.cancel_right` with `linear_equiv`s. In particular if `e` is a `linear_equiv` then `cancel_right e.surjective` doesn't work, but `cancel_right e.surjective_to_linear_map` does.

Reply:
Hope you can convince maintainers! In any case, it doesn't hurt to switch to shorter proofs. I'll take a look at #13728 where I think you (plan to) use these lemmas.